build/Dockerfile

```diff
@@ -1,26 +1,45 @@
-FROM centos:centos8
+FROM quay.io/centos/centos:stream8
 
 # Use this build arg to set any default test script arguments
-ARG RUN_SCRIPT_ARGS=
 ENV RUN_SCRIPT_ARGS=${RUN_SCRIPT_ARGS}
-
+ENV ROBOT_EXTRA_ARGS=${ROBOT_EXTRA_ARGS}
 ENV HOME /tmp
 
+## Install supporting libraries and driver
 RUN dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm &&\
-    dnf install -y git python3 unzip chromium chromedriver &&\
+    dnf install -y python38 jq git unzip chromium chromedriver redhat-lsb-core &&\
     dnf clean all
 
+## Install oc in the container
+RUN curl -L https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-linux.tar.gz -o $HOME/oc_client.tar.gz && \
+    tar xvf $HOME/oc_client.tar.gz -C /usr/local/bin/ && \
+    rm -rf $HOME/oc_client.tar.gz && rm /usr/local/bin/README.md
+
+RUN chmod 755 /usr/local/bin/oc &&\
+         oc version --client
+
+## Install OCM in the container
+RUN curl -L  https://github.com/openshift-online/ocm-cli/releases/download/v0.1.64/ocm-linux-amd64 -o /usr/local/bin/ocm
+RUN chmod 755 /usr/local/bin/ocm && ocm version
+
+# Create and Change the WORKDIR so the run script references any files/folders from the root of the repo
 RUN mkdir $HOME/rhoda-ci
```

Comment:
@mrrajan Could you please explain the code about creating the WORKDIR? Shouldn't we git clone rhoda-ci repo from github instead of creating a blank folder rhoda-ci?

Reply:
@harshkumarRH This is on containerization, we would clone the repo on our local and then the files from the local would be copied to generate an image. On the container, we are not directly cloning our framework. So whenever we have changes on our repo, we have to create a new image from local and push it to the Quay repository (we can think of automating this part)

Reply:
Thank you for explaining